tgui/.eslintrc.yml

```diff
@@ -708,11 +711,9 @@ rules:
   ## Enforce event handler naming conventions in JSX
   react/jsx-handler-names: error
   ## Validate JSX indentation (fixable)
-  react/jsx-indent: [error, 2, {
-    checkAttributes: true,
-  }]
+  # react/jsx-indent: [error, tab, { checkAttributes: true }]
   ## Validate props indentation in JSX (fixable)
-  react/jsx-indent-props: [error, 2]
+  react/jsx-indent-props: [error, tab]
```

Comment:
Disable all indent rules because now prettier handles it, less rules for eslint means it finishes linting wuicker.

Reply:
It does, and I can, but I was thinking we should leave some warnings on to let players know they're coding against standards (if they don't have prettier).
I can revert the previous commit for this